nucypher-core-python/stubtest-allowlist.txt

```diff
@@ -0,0 +1 @@
+nucypher_core.umbral.Parameters.__init__
```

Comment:
Pardon my ignorance, what's this for?

Reply:
We're using [`mypy.stubtest`](https://github.com/nucypher/nucypher-core/pull/73/files#diff-5e0f5577f0715558b9e0c39adb2d162e51c3f95ea9709aee682aab8a7c9cca65R128) to make sure `*.pyi` files match runtime of bindings generated by `pyo3`